python/ray/actor.py

```diff
@@ -398,6 +398,14 @@ def _remote(self,
             kwargs = {}
         if is_direct_call is None:
             is_direct_call = ray_constants.direct_call_enabled()
+
+        meta = self.__ray_metadata__
+        actor_has_async_methods = len(
+            inspect.getmembers(
+                meta.modified_class,
+                predicate=inspect.iscoroutinefunction)) > 0
+        is_asyncio = is_asyncio or actor_has_async_methods
```

Comment:
We should probably get rid of the `is_asyncio` flag altogethe? Or is there a reason to have the flag?

Reply:
Removed!